docs/model_library/strategic_water_management/index.rst

```diff
@@ -1451,51 +1451,51 @@ For :math:`t > 1`:
         + \sum_{k \in K | (s, k) \in SKT}\textcolor{purple}{F_{s,k,t}^{Trucked}}
         + \textcolor{purple}{F_{s,t}^{StorageEvaporationStream}})
 
-**Treatment Site Water Quality** :math:`\forall \textcolor{blue}{r \in R}, \textcolor{blue}{w \in W}, \textcolor{blue}{t \in T}`
+**Treatment Site Water Quality** :math:`\forall \textcolor{blue}{r \in R}, \textcolor{blue}{qc \in QC}, \textcolor{blue}{t \in T}`
 
 The water quality at treatment sites is dependent on the flow rates into the treatment site and the water quality of the flows. Even mixing is assumed, so all outgoing flows have the same water quality. The treatment process does not affect water quality.
```

Comment:
This part of the documentation should be updated - the model has changed, and the treatment process now does affect water quality.

Reply:
Updated!